concepts/structs/about.md

```diff
@@ -39,6 +41,37 @@ fmt.Printf("name: %s age: %d\n", person.name, person.age)
 // Output: name: age: 0
 ```
 
+Another way of creating a new instance of a struct is by using the `new` built-in: 
+```go
+s2 := new(Shape)
+fmt.Printf("name: %s size: %d\n", s2.name, s2.size)
+// Output: name:  size: 0
+```
+
+Sometimes it can be nice to have a _constructor_, which is a function to create an instance of a struct.
+The following constructor can be used to create a new instance of `Shape` and automatically set a default-value for the size of the shape:
+
+```go
+func NewShape(name string) Shape {
+	return Shape{
+		name: name,
+		size: 100, //default-value for size is 100
+	}
+}
+```
+
+Using a constructor can have the following advantages:
+* validation of the given values
+* handling of default-values
```

Comment:
```suggestion
Using `New` functions can have the following advantages:

* validation of the given values
* initialization of the struct fields with default values other than the zero value for the type
* since `New` functions are often declared in the same package of the structs they initialize, they can initialize even private fields of the struct
```